actor/builtin/miner/miner.go

```diff
@@ -34,15 +34,10 @@ func init() {
 // https://github.com/filecoin-project/specs/pull/318
 const LargestSectorSizeProvingPeriodBlocks = 1000
 
-// LargestSectorGenerationAttackThresholdBlocks defines the number of blocks
-// after a proving period ends after which a miner using the largest sector size
-// supported by the network is subject to storage fault slashing.
-//
-// TODO: If the following PR is merged - and the network doesn't define a
-// largest sector size - this constant and consensus.AncestorRoundsNeeded will
-// need to be reconsidered.
-// https://github.com/filecoin-project/specs/pull/318
-const LargestSectorGenerationAttackThresholdBlocks = 100
+// PoStChallengeTimeBlocks defines the blocks time prior to the proving
+// period end at which the PoSt challenge seed is chosen. This dictates the
+// earliest point at which a PoSt may be submitted.
+const PoStChallengeTimeBlocks = 500
```

Comment:
I think `PoStChallengePeriodBlocks` is slightly better. Or "window" instead of "period".